code/src/components/Header/Introduction.css

```diff
@@ -0,0 +1,51 @@
+.Introduction {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-content: center;
+    align-items: flex-start;
+    gap: 2rem;
+    margin: 45% 16px;
+    width: 60%;
+    position: relative;
+    top: 0;
+    z-index: 3;
+  }
+  
+  .mainbutton {
+    color: var(--main-dark-color);
+    background-color: var(--secondary-color);
+    padding: 8px 16px;
+    border-radius: 16px;
```

Comment:
Use the cursor: pointer; property for the .mainbutton class to change the cursor appearance when the user hovers over the button, indicating that it's clickable.